src/dreammaker/ast.rs

```diff
@@ -552,14 +552,13 @@ pub enum Term {
 
 impl Term {
     pub fn is_static(&self) -> bool {
-        return match self {
-            Term::Null |
-            Term::Int(_) |
-            Term::Float(_) |
+        matches!(self,
+            Term::Null      |
```

Comment:
not a fan of the alignment being used here and on the other `matches!`, not sure if that was you or the tool though

Reply:
It was me, how would you prefer it to look like?

Reply:
`|` either at the end of the line without spaces, or at the beginning of the following line...